ovs/action_test.go

```diff
@@ -535,6 +535,40 @@ func TestSetTunnel(t *testing.T) {
 	}
 }
 
+func TestMultipath(t *testing.T) {
+	var tests = []struct {
+		desc   string
+		a      Action
+		action string
+		err    error
+	}{
+		{
+			desc:   "set multipath OK",
+			a:      Multipath("symmetric_l3l4+udp", 1024, "hrw", 2, 0, "reg0"),
+			action: "multipath(symmetric_l3l4+udp,1024,hrw,2,0,reg0)",
+		},
```

Comment:
Are there any unhappy cases to test?

Reply:
That's a good question, and I was asking it to myself. The part i'm not clear about is what is the role we want this API to have. I was thinking that at the end of the day, if you put something bad there, openvswitch will return an error. Do we try to implement the same safeguard in the API that there is already in openvswitch and hope to get it right 🤔  Like do we want to check all the algorithm names, to check if the destination variable make sense, etc 🤔